### PR TITLE
Replace Windows window-like build emoji with real window emoji

### PR DESCRIPTION
**Pull request by adamscott**
🏁 ➡️ 🪟
👏👥👏👥👏👥👏👥👏👥👏👥👏👥👏👥👏👥👏👥👏👥👏👥👏👥👏👥👏👥👏

### CODE REVIEW

File: .github/workflows/runner.yml [hunk 1]
```diff
@@ -41,7 +41,7 @@ jobs:
 
   windows-build:
     if: ${{ vars.DISABLE_GODOT_CI == '' }}
-    name: 🏁 Windows
+    name: 🪟 Windows
```
Comment (aaronfranke) [inline .github/workflows/runner.yml:44]: I prefer the flag personally.
```suggestion
    name: 🏁 Windows
```

**Comment by adamscott** [comment]
Here's what it looks like.
image

**Comment by aaronfranke** [comment]
The cross does not show up on Firefox on macOS, it just has a wood frame with a glass pane:
Screenshot 2023-12-12 at 10 45 48 AM

**Comment by JoNax97** [comment]
I can't see the new emoji, I see the default "missing character" white rectangle.

**Comment by YuriSizov** [comment]
I think we didn't go for the obvious one because it's a relatively new addition and thus not supported on all platforms. So we picked from a very conservative list.

That said, I think there is a good reason to keep things as they are. Workflows tend to [linger in the list](https://github.com/godotengine/godot/actions), with old names never going away. So a small rename like that could end extra noise to the sidebar on that page. Which is not the end of the world, especially now that we have a single entry point for all workflows, but I would like to avoid frivolous renames like that nevertheless 🙃

File: .github/workflows/runner.yml (hunk 1 above)
Comment (adamscott) [reply .github/workflows/runner.yml:44]: I didn't know about that! Thanks, Aaron!

**Pull request closed without merging**